.github/workflows/publish-release.yml

```diff
@@ -37,6 +40,16 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18
+      - name: Bump base config
+        run: |
+          npm install @teknologi-umum/eslint-config-base@latest
+          npm version patch
+      - name: Commit lockfile
+        run: |
+          git config --global user.name 'elianiva'
+          git config --global user.email 'elianiva@users.noreply.github.com'
```

Comment:
Should be `${{ github.actor }}`. See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context